pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -4009,3 +4022,14 @@ var _ = Describe("Function getNumberOfPodInterfaces()", func() {
 		Expect(causes).To(BeEmpty())
 	})
 })
+
+func keepAutoattachPodInterfaceDefault(vmi *v1.VirtualMachineInstance) {
```

Comment:
I think you can inline this and use `_` for the `vmi` argument.
```
Entry("should accept vmi with autoattached interfaces as the default", func(_ *v1.VirtualMachineInstance){})
```

But I guess it is not needed if you use the prev suggestion.

Reply:
Done according this path, no need to give name to the variable at all